platforms/kubernetes/actions/sites/create.ts

```diff
@@ -117,12 +119,26 @@ export default async function newSite(
   const [secretEnvVar] = await Promise.all([
     secretEnvVarPromise,
     ...setupPromises,
+    corev1Api.createPersistentVolume({
+      metadata: { name: `pv-${siteNs}` },
+      spec: {
+        accessModes: ["ReadWriteMany"],
+        storageClassName: EFS_SC,
+        persistentVolumeReclaimPolicy: "Retain",
```

Comment:
Not sure if this should be `Retain` or not, I was afraid that if the PV was created and couldn't bind to a PVC, that it would delete the AP. Not sure if that is how that works.